src/test/java/uk/gov/nationalarchives/pdi/step/jena/model/CanSkipIT.java

```diff
@@ -57,18 +59,22 @@ public static void setup() throws KettleException {
     }
 
     @Test
-    public void can_create_rdf_type_statement(@TempDir final Path tempDir) throws KettleException, IOException {
-        final Path outputFilePath = tempDir.resolve("output.ttl");
+    public void skipProperties(@TempDir final Path tempDir) throws KettleException, IOException {
+        final Model expected = ModelFactory.createDefaultModel();
+        expected.add(
+                expected.createResource("http://example.com/s"),
+                expected.createProperty(RDF_NAMESPACE_IRI + "pSkipNo"),
```

Comment:
Use Jena vocabularies, see adamretter/kettle-jena-plugins#1.

Reply:
Actually `pSkipNo` should not be in the RDF namespace at all, so I have moved it to a custom namespace.